front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -193,5 +198,26 @@ export const orderSelectorList = (list: (string | null)[]) => {
     : list.sort();
 };
 
+export const sortSelectedModeCurves = (curvesList: ConditionalEffortCurve[]) => {
```

Comment:
Nice function ! Can you add a comment to explain how we sort the curves and some unit tests ? (unit tests will be quick to write, you can use empty curves and only change the conditions)